### PR TITLE
Add more grade and language enums

### PR DESCRIPTION
**Pull request by bimal125**
## Changes

* Add more grade and language enums

Mention related users here if any.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)
- [ ] translations

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,8 +18,11 @@ jobs:
         run: |
           TAG=$(echo $GITHUB_SHA | head -c7)
           IMAGE="docker.pkg.github.com/toggle-corp/kitab-server"
-          echo ::set-output name=tagged_image::${IMAGE}:${TAG}
-          echo ::set-output name=tag::${TAG}
+          #echo ::set-output name=tagged_image::${IMAGE}:${TAG}
+          #echo ::set-output name=tag::${TAG}
```
Comment (kopitek8) [inline .github/workflows/ci.yml]: Remove this (if not needed)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -18,8 +18,11 @@ jobs:
...
+          echo "{tagged_image}=${IMAGE}:${TAG}" >> $GITHUB_OUTPUT
+          echo "{tag}=${TAG}" >> $GITHUB_OUTPUT
```
Comment (kopitek8) [inline .github/workflows/ci.yml]: For research purposes, can you please share the documentation which shows this implementation?

**Review by kopitek8 — CHANGES_REQUESTED** [review]
LGTM

File: .github/workflows/ci.yml (hunk 2 above)
Comment (bimal125) [reply .github/workflows/ci.yml]: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

**Review by thenav56 — APPROVED** [review]

**Pull request merged**